docs/source/create/avatars/find-avatars.rst

```diff
@@ -46,7 +46,7 @@ Install the Avatar Exporter
 
 You need to install the extension for every Unity project that you have. Keep in mind, however, that you can import and export multiple avatars in a single Unity project.  
 
-1. Download the `avatar exporter <https://github.com/highfidelity/hifi/raw/57c3620587ae1ec4638a58988909b46602c14633/tools/unity-avatar-exporter/avatarExporter.unitypackage>`_ from High Fidelity. 
+1. Download the `avatar exporter <https://github.com/highfidelity/hifi/raw/57c3620587ae1ec4638a58988909b46602c14633/tools/unity-avatar-exporter/avatarExporter.unitypackage>`_ from Project Athena. 
```

Comment:
High Fidelity GitHub URL needs replacing.

Reply:
I thought I got that... weird.